.github/workflows/update-release.yaml

```
@@ -36,195 +39,31 @@ jobs:
-            echo "Error: linux_arm64 SHA256 not found in checksums file"
-            exit 1
-          fi
-          
-          echo "darwin_amd64_sha=$DARWIN_AMD64_SHA" >> $GITHUB_OUTPUT
-          echo "darwin_arm64_sha=$DARWIN_ARM64_SHA" >> $GITHUB_OUTPUT
-          echo "linux_amd64_sha=$LINUX_AMD64_SHA" >> $GITHUB_OUTPUT
-          echo "linux_arm64_sha=$LINUX_ARM64_SHA" >> $GITHUB_OUTPUT
-
       - name: Create branch
         id: branch
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:3:26: Double quote to prevent globbing and word splitting

(shellcheck)